src/libraries/System.Private.CoreLib/src/System/Numerics/Vector.cs

```diff
@@ -62,6 +63,17 @@ public static Vector<T> Abs<T>(Vector<T> value)
         [Intrinsic]
         public static Vector<T> Add<T>(Vector<T> left, Vector<T> right) => left + right;
 
+        /// <summary>Performs saturating addition on two vectors.</summary>
+        /// <param name="left">The vector to add with <paramref name="right" />.</param>
+        /// <param name="right">The vector to add with <paramref name="left" />.</param>
+        /// <typeparam name="T">The type of the elements in the vector.</typeparam>
+        /// <returns>The saturated sum of <paramref name="left" /> and <paramref name="right" />.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Vector<T> AddSaturate<T>(Vector<T> left, Vector<T> right)
+        {
+            return VectorMath.AddSaturate<Vector<T>, T>(left, right);
```

Comment:
There are intrinsics for `Vector<T>` in `System.Runtime.Intrinsics.Arm.Sve` but the class is marked as experimental, so i opted out of using them here.